internal/k8s/ingress.go

```diff
@@ -48,37 +49,83 @@ func Ingress(hosts []string) *networkingv1.Ingress {
 }
 
 // ingressRule creates a rule for the host with proper API routing.
-func ingressRule(host string) networkingv1.IngressRule {
-	var pathType = networkingv1.PathType("Prefix")
+func ingressRules(chartVersion string, host string) networkingv1.IngressRule {
+	rules := ingressRulesForV1()
+	if helm.ChartIsV2Plus(chartVersion) {
```

Comment:
For this we probably want to be checking against Airbyte versions? That's not something we currently do, or truly support.